provisions/roles/db/tasks/main.yml

```diff
@@ -39,8 +39,9 @@
 
 - name: Pull CentOS postgres container
   docker_image:
-      name: registry.centos.org/sclo/postgresql-95-centos7
+      name: '{{ postgresql_image }}'
       tag: latest
+      force: yes
```

Comment:
I think we'll need `state: present` as well @dharmit , since `force` functionality is clubbed with `state` config.
PTAL at [this](http://docs.ansible.com/ansible/latest/docker_image_module.html) doc, the force table entry.